### PR TITLE
kyverno-notation-aws package build

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [ ] This PR links to the upstream project's support policy (e.g. `endoflife.date`)

### CODE REVIEW

File: kyverno-notation-aws.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      modroot: .
+      output: kyverno-notation-aws
+      packages: .
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: "Kyverno extension service for Notation and the AWS signer (compatibility)"
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/
```
Comment (kranurag7) [inline kyverno-notation-aws.yaml:33]:
```suggestion

```
nit: we can omit this line given we're already creating a nested directory on the next line.

File: kyverno-notation-aws.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+          mkdir -p ${{targets.contextdir}}/plugins/com.amazonaws.signer.notation.plugin
+          ln -sf /usr/bin/kyverno-notation-aws ${{targets.contextdir}}/kyverno-notation-aws
+          ln -sf /usr/bin/notation-com.amazonaws.signer.notation.plugin ${{targets.contextdir}}/plugins/com.amazonaws.signer.notation.plugin/notation-com.amazonaws.signer.notation.plugin
```
Comment (kranurag7) [inline kyverno-notation-aws.yaml:36]: should we include https://github.com/wolfi-dev/os/blob/main/aws-signer-notation-plugin.yaml as a runtime dependency to this subpackage if they're related else we can pull in image without being specific to a version which is always latest.

Comment (developer-guy) [reply kyverno-notation-aws.yaml:36]: I can do that but I was planning to use the same -compat sub-package for FIPS image, this is why I didn't specify it as a runtime dependency, does that make sense?

Comment (kranurag7) [reply kyverno-notation-aws.yaml:36]: perfect that looks good to me.

**Review by kranurag7 — APPROVED** [review]
thanks for being patient with reviews. Looks good.

File: kyverno-notation-aws.yaml (hunk 1 above)
Comment (developer-guy) [reply kyverno-notation-aws.yaml]: removed brother

**Pull request merged**